cmd/snap/cmd_run.go

```diff
@@ -35,11 +36,14 @@ import (
 	"github.com/snapcore/snapd/osutil"
 	"github.com/snapcore/snapd/snap"
 	"github.com/snapcore/snapd/snap/snapenv"
+	"github.com/snapcore/snapd/x11"
 )
 
 var (
 	syscallExec = syscall.Exec
 	userCurrent = user.Current
+	getEnv = os.Getenv
```

Comment:
(Super nicktpick) most code uses the convention `osGetenv = os.Getenv` (same in the line below).